community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+## Basic Protocol in case you are accused of harassment/bullying/offending/doing weird shits/unacceptable behaviour
+
+Subject -> Who has been accused
+
+1. Well the subject can start by saying sorry, and ensuring that similar behaviour is not repeated in front of the person who has accused the subject "ever". If it's ingrained in the subject’s habit let the accussee know that, it's in your habit you will make sincere effort to decrease the frequency. The subject can take a note and share updates of progress {pronoun} have made on that person's feedback.
+2. Adding "genuinely" prefix in front of sorry doesn't help.
+3. It doesn't matter if the subject’s sorry or change is genuine or fake, as long as it's change it's good.
+
+## Basic Protocol in case you have noticed harassment.
+
+**@team comments**
+
+
+
+## Basic Protocol in case you are an executive/advisor and have to take action
+
+**@team comments**
+
+## What is Considered as Sexual Harassment at Work Place ?
+
+What workplace at koss includes, **@team ????????????**
+
+Sexual harassment includes many things but is not limited to
+
+[Ref](https://www.un.org/womenwatch/osagi/pdf/whatissh.pdf)
+
+1. Actual or attempted rape or sexual assault.
+2. Unwanted pressure for sexual favors.
+3. Unwanted deliberate touching, leaning over, cornering, or pinching.
+4. Unwanted sexual looks or gestures.
+5. Unwanted letters, telephone calls, or materials of a sexual nature.
+6. Unwanted pressure for dates.
+7. Unwanted sexual teasing, jokes, remarks, or questions.
+8. Referring to an adult as a girl, hunk, doll, babe, or honey.
+9. Whistling at someone.
+10. Cat calls.
+11. Sexual comments.
+12. Turning work discussions to sexual topics.
+13. Sexual innuendos or stories.
+14. Asking about sexual fantasies, preferences, or history.
+15. Personal questions about social or sexual life.
+16. Sexual comments about a person's clothing, anatomy, or looks.
+17. Kissing sounds, howling, and smacking lips.
+18. Telling lies or spreading rumors about a person's personal sex life.
+19. Neck massage.
+20. Touching an employee's clothing, hair, or body.
+21. Giving personal gifts., Hanging around a person.
+22. Hugging, kissing, patting, or stroking.
+23. Touching or rubbing oneself sexually around another person.
+24. Standing close or brushing up against a person.
+25. Looking a person up and down (elevator eyes).
+26. Staring at someone.
+27. Sexually suggestive signals.
+28. Facial expressions, winking, throwing kisses, or licking lips.
+29. Making sexual gestures with hands or through body movements.
+
+
+
+The point of having this list, is to make people aware that these things can be considered offensive and harassment, by some people. We come from different culture, for example hugging might be quite common in a few culture, but might not be well received by some other culture. So asking “if a person is comfortable” might help in all verbal/non-verbal/physical contact.
+
+## What is considered as bullying ?
```

Comment:
Should we have bullying in this code of conduct for sexual harassment or should it be a separate doc instead?

Reply:
TBH, I don't know. I have ping-ponged this thought myself to keep them together or not, some reasons to support why to keep them together :-
1. I believe bullying/sexual-harassment are fundamentally quite similar, one guideline will fit other.
2. Easy to maintain
I don't have enough reasons to keep them apart, apart from my own vanity.

Reply:
We may want to add that if something is making a person uncomfortable and is not included in the list or even not thought of as harassment, they should come forward and discuss to help KOSS be more inclusive and comfortable. Just the thought of discomfort with a person or situation should be addressed otherwise it leads to the individual separating from the organization.